src/librustc_mir/borrow_check/nll/mod.rs

```diff
@@ -201,9 +202,9 @@ pub(in borrow_check) fn compute_regions<'cx, 'gcx, 'tcx>(
     (regioncx, polonius_output, closure_region_requirements)
 }
 
-fn dump_mir_results<'a, 'gcx, 'tcx>(
+fn dump_mir_results<'a, 'gcx, 'tcx, V: Idx>(
     infcx: &InferCtxt<'a, 'gcx, 'tcx>,
-    liveness: &LivenessResults,
+    liveness: &LivenessResults<V>,
```

Comment:
...but instead here have `LivenessResults<Local>` (at least to start -- later we'll change this I think)